packages/now-cli/test/dev-validate.unit.js

```diff
@@ -68,10 +68,7 @@ test('[dev-validate] should error with invalid routes array type', async t => {
     routes: { src: '/(.*)', dest: '/api/index.js' },
   };
   const error = validateConfig(config);
-  t.deepEqual(
-    error.message,
-    'Invalid vercel.json - property `routes` should be of type array.'
-  );
+  t.deepEqual(error.message, 'Invalid vercel.json - `routes` should be array.');
```

Comment:
All of these "should be" errors would read better to me if we prefixed "a"/"an" before the type. i.e. "should be _an_ array".